public/locations-list-map.js

```diff
@@ -636,11 +631,21 @@ window.showMarkers = showMarkers; // Exposed for debug/testing.
 
 
 // TODO (patricknelson): Some of this is duplicated in centerMapToMarkersNearCoords(). Need issue/PR to discuss how to de-duplicate.
+const MAP_INITIAL_VIEW = {
+  ca: { zoom: 4, center:{ lat: 39.0567939, lng: -94.6065124 }},
+  ch: { zoom: 4, center:{ lat: 39.0567939, lng: -94.6065124 }},
+  de: { zoom: 4, center:{ lat: 39.0567939, lng: -94.6065124 }},
+  gb: { zoom: 4, center:{ lat: 39.0567939, lng: -94.6065124 }},
+  fr: { zoom: 4, center:{ lat: 39.0567939, lng: -94.6065124 }},
+  us: { zoom: 4, center:{ lat: 39.0567939, lng: -94.6065124 }},
```

Comment:
These are all the same - separate task to figure out proper center+zoom per country?

Reply:
sorry, wasn't ready fore view.. check again.